Mathlib/Analysis/Normed/Group/Basic.lean

```diff
@@ -935,6 +968,16 @@ theorem mem_closure_one_iff_norm {x : E} : x ∈ closure ({1} : Set E) ↔ ‖x
 theorem closure_one_eq : closure ({1} : Set E) = { x | ‖x‖ = 0 } :=
   Set.ext fun _x => mem_closure_one_iff_norm
 
+section Instances
+
+instance [SeminormedGroup E] : ContinuousENorm E where
+  continuous_enorm := continuous_enorm'--ENNReal.continuous_coe.comp continuous_nnnorm'
+
+instance [SeminormedAddGroup E] : ContinuousENorm E where
+  continuous_enorm := ENNReal.continuous_coe.comp continuous_nnnorm'
```

Comment:
Another fun fact: `continuous_enorm` does *not* work here - while `continuous_enorm'` works three lines above.

Reply:
That seems like a serious issue. Did you investigate?

Reply:
To the extent of "this seems wrong, but I don't really know why/how to narrow this down", yes. Put differently: I'm asking you to better understand what's going on here, and your last answers are certainly progress.
I don't understand yet why this is. I'll tell you if the most recent fix helps (after bumping carleson).

Reply:
The error here is that you have both `SeminormedGroup E` (as a section variable) and `SeminormedAddGroup E`, so you have two different norms.